### PR TITLE
Adding ultranest package

### PR DESCRIPTION
**Pull request by JohannesBuchner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

- python/c hybrid `@conda-forge/help-python-c`

### CODE REVIEW

**Comment by JohannesBuchner** [comment]
There seems to be an issue with AppVeyor's SSL verification. I am not sure if I should fix that.

Why is AppVeyor running python 2.7 even though I specified to skip python<3.0?

@conda-forge/help-python-c

**Comment by JohannesBuchner** [comment]
@conda-forge-admin, please restart ci

**Comment by JohannesBuchner** [comment]
@conda-forge-admin, please rerender

**Comment by xhochy** [comment]
> @conda-forge-admin, please rerender

This only works on feedstocks.

File: .appveyor.yml [hunk 1]
```diff
@@ -18,10 +18,10 @@ platform:
 
 install:
     # If there is a newer build queued for the same PR, cancel this one.
-    - cmd: |
-        powershell -Command "(New-Object Net.WebClient).DownloadFile('https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/master/recipe/ff_ci_pr_build.py', 'ff_ci_pr_build.py')"
-        ff_ci_pr_build -v --ci "appveyor" "%APPVEYOR_ACCOUNT_NAME%/%APPVEYOR_PROJECT_SLUG%" "%APPVEYOR_BUILD_NUMBER%" "%APPVEYOR_PULL_REQUEST_NUMBER%"
-        del ff_ci_pr_build.py
+    #- cmd: |
```
Comment (xhochy) [inline .appveyor.yml]: Please revert

File: conda-forge.yml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 max_py_ver: '37'
 max_r_ver: '35'
+provider:
```
Comment (xhochy) [inline conda-forge.yml]: Please revert

File: recipes/ultranest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (xhochy) [inline recipes/ultranest/meta.yaml]: Remove the comments

**Comment by JohannesBuchner** [comment]
I don't know why this worked now, but it worked.

**Pull request merged**